rotkehlchen/types.py

```diff
@@ -597,6 +598,24 @@ def to_chain_id(self) -> int:
 EVM_LOCATIONS: tuple[EVM_LOCATIONS_TYPE_, ...] = typing.get_args(EVM_LOCATIONS_TYPE_)
 
 
+class LocationDetails(NamedTuple):
+    display_name: Optional[str] = None
+    icon: Optional[str] = None
+    image: Optional[str] = None
+
+    def serialize(self) -> dict[str, str]:
```

Comment:
you are not using asdict here only to have optional stuff not appear?

Reply:
Yes, the logic using asdict would be similar but using pop with if checks for None